src/websocket.zig

```diff
@@ -5,7 +5,9 @@
 // -------------------------------------------------------------------------- //
 const std = @import("std");
 const web = @import("zhp.zig");
+const util = @import("util.zig");
 const log = std.log;
+const native_endian = util.native_endian;
```

Comment:
Minor nitpick but I'd prefer just importing from std again here unless there's some benefit of adding it to `util.zig`